<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="WIN-22PUACE5BKI" failures="1" tests="5" name="Test_assignment.Demchenko.Tests" time="37.910" errors="0" timestamp="2024-01-24T18:38:24 EET" skipped="0">
  <testcase classname="Test_assignment.Demchenko.Tests" name="HomePageIsOpened" time="3.338"/>
  <system-out/>
  <testcase classname="Test_assignment.Demchenko.Tests" name="QAPositionsListPresenceWithFiltration" time="9.461"/>
  <system-out/>
  <testcase classname="Test_assignment.Demchenko.Tests" name="CheckFilteredPositions" time="10.022">
    <failure message="expected:&lt;Istanbul, Turkey&gt; but was:&lt;&gt;" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected:<Istanbul, Turkey> but was:<>
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:82)
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:87)
at Test_assignment.Demchenko.Tests.CheckFilteredPositions(Tests.java:65)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:425)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:68)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
]]>
    </failure>
  </testcase> <!-- CheckFilteredPositions -->
  <system-out/>
  <testcase classname="Test_assignment.Demchenko.Tests" name="CareersPageBlocksAreOpened" time="6.153"/>
  <system-out/>
  <testcase classname="Test_assignment.Demchenko.Tests" name="RedirectionToPositionPage" time="8.936"/>
  <system-out/>
</testsuite> <!-- Test_assignment.Demchenko.Tests -->
